test/e2e/launch.js

```diff
@@ -11,9 +11,10 @@ const shell = require(`shelljs`)
 
 const networkPath = join(__dirname, "localtestnet")
 
-let app, home, nodeHome, started
-let binary = process.env.BINARY_PATH || "$GOPATH/bin/gaiacli"
-let nodeBinary = process.env.NODE_BINARY_PATH
+let app, cliHome, nodeHome, started
+let binary = process.env.BINARY_PATH || process.env.GOPATH + "/bin/gaiacli"
```

Comment:
for my own learning, why is it better to use `process.env.GOPATH` over `$GOPATH`?

Reply:
`$GOPATH` is not necessarily resolved